packages/flutter/test/material/floating_action_button_test.dart

```diff
@@ -687,20 +704,26 @@ void main() {
   });
 
   testWidgets('Floating Action Button has no clip by default', (WidgetTester tester) async {
+    final FocusNode focusNode = FocusNode();
     await tester.pumpWidget(
       Directionality(
-          textDirection: TextDirection.ltr,
-          child: Material(
-            child: FloatingActionButton(
-              onPressed: () { /* to make sure the button is enabled */ },
-            ),
+        textDirection: TextDirection.ltr,
+        child: Material(
+          child: FloatingActionButton(
+            focusNode: focusNode,
+            clipBehavior: Clip.none,
```

Comment:
Doesn't this defeat the purpose of this test? It is no longer default if you are forcing it to `Clip.none`.

Reply:
Duh.  I must have been testing a theory... Removed.